jupyter_client/kernelspecapp.py

```diff
@@ -138,6 +144,61 @@ def start(self):
                 self.exit(1)
             raise
 
+class RemoveKernelSpec(JupyterApp):
+    version = __version__
+    description = """Remove one or more Jupyter kernelspecs by name."""
+    examples = """jupyter kernelspec remove python2 [my_kernel ...]"""
+    
+    force = Bool(False, config=True,
+        help="""Force removal, don't prompt for confirmation."""
+    )
+    spec_names = List(Unicode())
+    
+    kernel_spec_manager = Instance(KernelSpecManager)
+    def _kernel_spec_manager_default(self):
+        return KernelSpecManager(data_dir=self.data_dir, parent=self)
+    
+    flags = {
+        'f': ({'RemoveKernelSpec': {'force': True}}, force.metadata['help']),
```

Comment:
that's exactly what it is. I saw this earlier, but forgot to push the fix, sorry. Should be okay, now.